tests/lean/weirdmacro.lean

```diff
@@ -1,4 +0,0 @@
-macro term x:term : term => `($x)
```

Comment:
@leodemoura I removed this test case since the leading `term` would now be interpreted as the actual parser. This test case only existed to check for absence of the previous "auto-quoting" feature, I don't think any user would write down something like this and expect something sensible to happen.